src/operations/indexes.ts

```diff
@@ -214,6 +214,18 @@ function resolveIndexDescription(
   );
 }
 
+/**
+ * @public
+ * The index information returned by the listIndexes command. https://www.mongodb.com/docs/manual/reference/command/listIndexes/#mongodb-dbcommand-dbcmd.listIndexes
+ */
+export type IndexDescriptionInfo = Omit<IndexDescription, 'key' | 'version'> & {
+  key: { [key: string]: IndexDirection };
+  v?: IndexDescription['version'];
+};
+
```

Comment:
We generally try to keep our type definitions for server responses flexible so that if the server modifies a response in a future server version the driver is still compatible with it.  Could we `& Document` here?  I think that will provide type safety on known keys but will allow unknown keys without a Typescript error.